product_pricelist_assortment/models/product_pricelist_assortment_item.py

```diff
@@ -14,6 +14,30 @@ class ProductPricelistAssortmentItem(models.Model):
     _description = "Product Pricelist Assortment Item"
     _inherit = "product.pricelist.item"
 
+    base = fields.Selection(
```

Comment:
This is not a view change.

Reply:
Yes it is not. Whithout it transaltion does not work.

Reply:
I don't get it.

Reply:
You can see on screenshots. Values of selection fields was not transalted before.

Reply:
They won't be translated because the source was not, but it's not something you should "fix" here repeating the field definition.

Reply:
ok got it